### PR TITLE
Update scan

### PR DESCRIPTION
**Pull request by nerya10**

### CODE REVIEW

File: cli/flows-cli.sh [hunk 1]
```diff
@@ -48,10 +48,36 @@ prereq_check() {
...
+  if [[ $exit_code -eq 143 || $exit_code -eq 130 ]]; then
     echo "Script was terminated by user. Exit code: $exit_code"
+    update_scan_status "canceled"
   else
     echo "An error occurred. Exit code: $exit_code"
+    update_scan_status "failed"
+  fi
+}
+
+update_scan_status() {
```
Comment (david-piiano) [inline cli/flows-cli.sh:60]: We should turn this API to talk to the backend to a real function and reuse twice

Comment (nerya10) [reply cli/flows-cli.sh:60]: Can you explain?

**Review by david-piiano — APPROVED** [review]

File: cli/flows-cli.sh [hunk 2]
```diff
@@ -48,13 +49,44 @@ prereq_check() {
...
+validate_response() {
+  local response="$1"
+  http_status=$(echo "$response" | grep -Fi HTTP/ | awk '{print $2}')
+  body=$(echo "$response" | sed '1,/^\r$/d')
+
+  if [ "$http_status" != "200" ]; then
+      echo "[ ] Error: ${body}"
+      exit 1
+  fi
+  echo "$body"
```
Comment (david-piiano) [inline cli/flows-cli.sh:87]: Do you need that or a leftover?

Comment (nerya10) [reply cli/flows-cli.sh:87]: This is how we return the body, no ?

**Pull request merged**